packages/boxed-expression-component/src/table/BeeTable/BeeTableEditableCellContent.tsx

```diff
@@ -74,6 +78,12 @@ export function BeeTableEditableCellContent({
     setEditingValue((prev) => (isEditing ? prev : value));
   }, [isEditing, value]);
 
+  useEffect(() => {
+    if (isActive) {
+      beeGwtService?.selectObject(cellId);
+    }
+  }, [isActive, beeGwtService, cellId]);
+
```

Comment:
We can move this up to the `BeeTableDefaultCell`.. this way we avoid an extra prop here :)

Reply:
@tiagobento Changes applied!